build.gradle

```
@@ -65,6 +65,8 @@ sourceSets {
                 exclude "stirling/software/SPDF/model/SessionEntity.java"
                 exclude "stirling/software/SPDF/model/User.java"
                 exclude "stirling/software/SPDF/repository/**"
+            } else {
+                exclude "stirling/software/SPDF/config/anonymus/**"
```

I think you have a typo here:
```suggestion
                exclude "stirling/software/SPDF/config/anonymous/**"
```